src/templates/activityTemplate/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Layout from '../../layouts/default';
+import mockHero from '../../images/mockHero.png';
+import HeroTitleSection from '../../components/heroTitleSection';
+import { PaddedContainer } from '../../components/ui.general';
+import ActivityContentSection from './activityContentSection';
+import AdditionalInfoSection from './additionalInfoSection';
+import ActivitySpecsSection from './activitySpecsSection';
+import { ActivityPageData } from './types';
+
+interface ActivityPageTemplateProps {
+  pageContext: {
+    data: ActivityPageData;
+  };
+}
+
+const ActivityPageTemplate = ({ pageContext }: ActivityPageTemplateProps) => (
```

Comment:
I would suggest the following typings instead. Then all of the Gatsby's page data is typed correctly.

```suggestion
import { PageProps } from 'gatsby';

interface ActivityPageTemplateContext {
  data: ActivityPageData;
}

const ActivityPageTemplate = ({ pageContext }: PageProps<object, ActivityPageTemplateContext>) => (
```